src/Clash/Cores/ClashFloPoCo/GenTemDSL.hs

```diff
@@ -0,0 +1,1306 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# OPTIONS_GHC -Wno-unused-binds #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+{-# OPTIONS_HADDOCK show-extensions #-}
+
+-- |
+--  Copyright   :  (C) 2024, Hoang Minh Le <minhxecole@gmail.com>
+--  License     :  BSD2
+--  Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>
+--  This module contains haskell templatefunction to generate BlackBoxFunction,
+--  BlackBoxTemplateFunction, TemplateFunction, Primitive and InfoEntity of FloPoCo arithmetic operations
+module Clash.Cores.ClashFloPoCo.GenTemDSL
+  ( getPipeDep,
+    flopocoPrim,
+    genFloPoCoInfoEntity,
+    liftInfoEntity,
+    genBlackBoxFunction,
+    genTemplateFunction,
+    genBlackBoxTemplateFunction,
+    genBlackBoxTemplateFunctionProd,
+    genBlackBox,
+    genBlackBoxProd,
+  )

```

Comment:
Use leading `,` for the export list, see for example: https://github.com/clash-lang/clash-cores/blob/3cc8c715cc905d36f562357ba6a1b35ea6e255c0/src/Clash/Cores/Xilinx/Floating.hs#L45-L76